src/Resources/Locales/de_DE.axaml

```diff
@@ -286,9 +286,9 @@
   <x:String x:Key="Text.GitLFS.Push.Title" xml:space="preserve">LFS Objekte pushen</x:String>
   <x:String x:Key="Text.GitLFS.Push.Tips" xml:space="preserve">Pushe große Dateien in der Warteschlange zum Git LFS Endpunkt</x:String>
   <x:String x:Key="Text.GitLFS.Remote" xml:space="preserve">Remote:</x:String>
-  <x:String x:Key="Text.GitLFS.Track" xml:space="preserve">Verfolge Dateien names '{0}'</x:String>
+  <x:String x:Key="Text.GitLFS.Track" xml:space="preserve">Verfolge '{0}' benannte Dateien</x:String>
   <x:String x:Key="Text.GitLFS.TrackByExtension" xml:space="preserve">Verfolge alle *{0} Dateien</x:String>
-  <x:String x:Key="Text.Histories" xml:space="preserve">Historie</x:String>
+  <x:String x:Key="Text.Histories" xml:space="preserve">Historien</x:String>
```

Comment:
I know it's plural in English, but I'm not sold on that choice. To me there is only one history which contains multiple branches, instead of many histories of multiple branches.

"Historien" is also something that is more linked to [literature/theatre according to dict.cc](https://www.dict.cc/?s=historien), which is also my gutfeeling.

Reply:
What about "**(Commit-)Verlauf**"? "Verlauf" is a very common wording used in browsers.
Or "Alle Commits".

Reply:
> I know it's plural in English, but I'm not sold on that choice. To me there is only one history which contains multiple branches, instead of many histories of multiple branches.
> 
> "Historien" is also something that is more linked to [literature/theatre according to dict.cc](https://www.dict.cc/?s=historien), which is also my gutfeeling.

Yeah you're right. Didn't extensively think about this one but one history does indeed sound much more logical. I prefer just "Verlauf" over "Commitveraluf" so would go with that.